R/xl_open.R

```diff
@@ -1,23 +1,26 @@
-#' @name xl_open
-#' @title Open a Microsoft Excel file (xls/xlsx) or an openxlsx Workbook
-#' @description This function tries to open a Microsoft Excel
-#' (xls/xlsx) file or an openxlsx Workbook with the proper
-#' application, in a portable manner.
+#' Open a Microsoft Excel file (xls/xlsx) or an openxlsx Workbook
 #'
-#' In Windows (c) and Mac (c), it uses system default handlers,
-#' given the file type.
+#' @description This function tries to open a Microsoft Excel (xls/xlsx) file or
+#'   an openxlsx Workbook with the proper application, in a portable manner.
 #'
-#' In Linux it searches (via `which`) for available xls/xlsx
-#' reader applications (unless `options('openxlsx.excelApp')`
-#' is set to the app bin path), and if it finds anything, sets
-#' `options('openxlsx.excelApp')` to the program chosen by
-#' the user via a menu (if many are present, otherwise it will
-#' set the only available). Currently searched for apps are
-#' Libreoffice/Openoffice (`soffice` bin), Gnumeric
-#' (`gnumeric`) and Calligra Sheets (`calligrasheets`).
+#'   In Windows it uses `base::shell.exec()` (Windwos only function) to
+#'   determine the appropriate program.
 #'
-#' @param file path to the Excel (xls/xlsx) file or Workbook object.
-#' @usage xl_open(file=NULL)
+#'   In Mac (c) it uses system default handlers, given the file type.
```

Comment:
I'm not sure, I will test later, but it should be possible to open workbooks with non default system handlers. Aka you could force opening the file with Numbers, but I haven't actually tried this.

Reply:
On mac the following works: 
```bash
open -a "Numbers" filepath/filename
```